### PR TITLE
[Live-Site] Feature/loading

### PR DESCRIPTION
**Pull request by Pratiyushkumar**
### Issue
- #615 

### What is the change?
There should be a loader shown while kicking out the peers from a live stream, so when we kick peers, there is a delay between the request sent to the backend and the response from the backend, so to cover that delay we are putting a loader.

### Is Development Tested?

- [x] Yes
- [ ] No

### Change video recording

https://github.com/Real-Dev-Squad/website-www/assets/82165483/fbf43083-4463-4c1d-9a14-16a8142f1bfa

### CODE REVIEW

File: app/components/live-participants.hbs [hunk 1]
```diff
@@ -21,19 +21,33 @@
...
-                <IconifyIcon
-                  data-test-icon='remove'
-                  @icon='material-symbols:person-remove'
-                  width='36'
-                  height='36'
-                  color='red'
-                />
-              </button>
+              {{#if this.liveService.isUserRemoved}}
+                <div class='loading-container'>
```
Comment (prerana1821) [inline app/components/live-participants.hbs:25]: Please follow the BEM convention for the class

File: app/styles/live-sidebar.module.css [hunk 2]
```diff
@@ -141,6 +140,55 @@
...
 
+.loading-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.loading span {
+  width: 8px;
+  height: 8px;
+  margin: 0 5px;
```
Comment (prerana1821) [inline app/styles/live-sidebar.module.css:151]: Please use rem for consistency.

**Review by prerana1821 — CHANGES_REQUESTED** [review]
Hey @Pratiyushkumar,
image
I don't think we need both loading text and the animation. 
You can remove the "Loading" text. Also, can the space between the dots decrease a little?

File: app/components/live-participants.hbs (hunk 1 above)
Comment (Pratiyushkumar) [reply app/components/live-participants.hbs]: removed

File: app/styles/live-sidebar.module.css (hunk 2 above)
Comment (Pratiyushkumar) [reply app/styles/live-sidebar.module.css]: done

**Comment by Pratiyushkumar** [comment]
> Hey @Pratiyushkumar, image I don't think we need both loading text and the animation. You can remove the "Loading" text. Also, can the space between the dots decrease a little?

removed loading text only animation is there

**Comment by satyam73** [comment]
The loader is overall goo can we decrease the size of bubbles, a little and make it pink color(for all the three bubbles).

**Review by satyam73 — DISMISSED** [review]
LGTM

**Review by vinit717 — DISMISSED** [review]

**Review by satyam73 — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

File: app/components/live-participants.hbs [hunk 3]
```diff
@@ -24,20 +24,30 @@
...
-                    height='36'
-                    color='red'
-                  />
-                </button>
+                {{#if this.liveService.isUserRemoved}}
+                  <div class='loading'>
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                  </div>
```
Comment (prakashchoudhary07) [inline app/components/live-participants.hbs:32]: NIT: this can be broken into a separate component

**Pull request merged**